spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'User view', type: :feature do
```

Comment:
* Kindly add the following missing tests as they are required 
* `Tests that asserts you can see the user's profile picture.`
* `Tests that asserts you can see the user's first 3 posts.`
* `Tests that redirects to the post's show page when you click on a user's post.`